tests/queries/models.py

```diff
@@ -737,3 +737,8 @@ class RelatedIndividual(models.Model):
 
     class Meta:
         db_table = 'RelatedIndividual'
+
+
+class Ticket27397(models.Model):
```

Comment:
i'm not sure,for choose existing models this model is good https://github.com/django/django/blob/master/tests/queries/models.py#L611 `Order` ?